core/src/main/java/org/apache/calcite/sql/fun/SqlStdOperatorTable.java

```diff
@@ -2069,6 +2069,15 @@ public class SqlStdOperatorTable extends ReflectiveSqlOperatorTable {
   public static final SqlDatePartFunction SECOND =
       new SqlDatePartFunction("SECOND", TimeUnit.SECOND);
 
+  /**
+   * The SQL <code>MICROSECOND</code> operator. Returns the Second
```

Comment:
It should be returns Microseconds, not second

Reply:
@dm-shantanuc i had resolved this.Please have a look